### DISCUSSION TITLE
Possible to pre install or pack/addon?

### DISCUSSION THREAD

**Discussion by jimmykingss** [LegacyUpdate/LegacyUpdate#137] [Ideas]
I wonder if it is possible to get somehow legacy update preinstalled inside of an iso or how you can make an IE8 pack/addon and then add it in with nlite if you don't know what I'm talking about then visit the link also not sure if this should be in questions or not 

https://www.wincert.net/forum/topic/6897-tool-dxs-universal-updatepackaddon-creator/

**Comment by techguy16** [comment]
[Legacy Update nLite Addon.zip](https://github.com/kirb/LegacyUpdate/files/11070747/Legacy.Update.nLite.Addon.zip)
Use it to your delight, and share it with who you like, just cite your sources :smile: 

(Be aware that the install window will appear after installation, so you must have Ethernet drivers installed and an Ethernet cable plugged in)

**Reply by jimmykingss** [reply]
oh ok thanks

**Reply by techguy16** [reply]
I've made a whole repo with nLite Addons if you want more to integrate: [https://github.com/techguy16/nLiteAddons/releases?page=1](https://github.com/techguy16/nLiteAddons/releases?page=1)
